### ISSUE TITLE
Currencies on funder pages are always GBP

### ISSUE THREAD

**Issue by robredpath** [ThreeSixtyGiving/grantnav#420]
On funder pages, any amount of money is displayed as GBP on account of the JS table not having currency information, and the summary box assuming GBP

The summary box we're dealing with in #387. 

We should look at a way to make funder pages not incorrect. The minimal implementation would be to just remove currency symbols from the table, but let's see if we can make a nicer solution.

**Comment by robredpath** [comment]
@stevieflow this is the issue for funder pages

**Comment by stevieflow** [comment]
Great, thanks @robredpath

**Comment by morchickit** [comment]
Most of it is in GBP, so maybe we can just flag specific datasets that do distribute grant in different currencies?

**Comment by robredpath** [comment]
@kindly could we use the same sort of logic as we do on the search result pages for working out what the most common currency is? Can we set the currency for the tables at a page level?

**Comment by morchickit** [comment]
Didn't we fix it already @robredpath ?

**Comment by robredpath** [comment]
@morchickit we excluded non-GBP grants from those pages so that they were no longer misrepresenting grants that have been made. We haven't made the funder pages show grants in currencies other than GBP. 

To see this, for example, http://grantnav.threesixtygiving.org/grant/360G-SamworthFdn-001-124 is a USD grant that doesn't appear on their funder page - http://grantnav.threesixtygiving.org/funder/GB-CHC-265647 . This results in some slightly odd behaviour on that latter page - if you scroll the Recipients table all the way to the bottom, you see "58 to 69 of 69" top-right of the table, as the total is still correct. 

It's not a perfect fix, but we figured it was better than misrepresenting grants, and was what was relatively straightforward to do.
